students/Chris_Kenyon/Lesson_3/Mailroom_Pt1.py

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env python3
+
+# Lesson_3 Activity 4 Mailroom Part 1
+
+
+def page_break():
+    """ Print a separator to distinguish new 'pages'"""
+    print("_"*75+"\n")
+
+
+def menu_page():
+    """ Main Menu Page """
+    while True:
+        try:
+            print("Please choose one of the following options(1,2,3):"
+                  "\n1. Send a Thank you. \n2. Create a report\n3. Quit")
+            option = int(input('--->'))
+        except ValueError:
+            print("you have made an invalid choice, try again.")
+            page_break()
+            continue
+        if option < 1 or option > 3:
+            print("you have made an invalid choice, try again.")
+            page_break()
+            continue
+        return option
+
+
+def send_thanks():
+    """ Send Thanks """
+    page_break()
+    while True:
+        list_names = [item[0] for item in donor_chart]
+        amount = []
```

Comment:
Doesn't look like you need to assign this here, since you're reassigning to the input value below